### PR TITLE
Implement accessibility feedback

### PR DESCRIPTION
**Pull request by HSalmon**
Implementation of: https://trello.com/b/skuMQvLJ/loomio-sprint-board

### CODE REVIEW

**Comment by HSalmon** [comment]
@rdbartlett, @robguthrie, would love to get your eyes at some point? Or whatever you think is required

File: angular/core/components/thread_page/activity_card/activity_card.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 %section.activity-card{aria-labelledby: "activity-card-title"}
   %h2.lmo-card-heading#activity-card-title{translate: "discussion.activity"}
-  %a.activity-card__load-backwards{href: "", ng-show: "canLoadBackwards()", ng-click: "loadEventsBackwards()"}
+  %a.activity-card__load-backwards{href: "", ng-show: "canLoadBackwards()", ng-click: "loadEventsBackwards()", tabindex: "0"}
```
Comment (rdbartlett) [inline angular/core/components/thread_page/activity_card/activity_card.haml]: what other tabindex's do we have on this page? is 0 definitely the right index?

**Comment by rdbartlett** [comment]
Code looks fine. Rob are you happy to smoke this through your screenreader to QA?

File: angular/core/components/thread_page/activity_card/activity_card.haml (hunk 1 above)
Comment (robguthrie) [inline angular/core/components/thread_page/activity_card/activity_card.haml]: from memory, tabindex 0 is almost always the right one.

**Comment by robguthrie** [comment]
yes

**Comment by HSalmon** [comment]
@gdpelican, @robguthrie, would love to move this along today

**Comment by robguthrie** [comment]
I'll smoke test later today, after I get metabase upgrade done

**Comment by gdpelican** [comment]
I'm going to merge this and take the heat later if it's bad (looks low risk to me). :D

**Pull request merged**